src/rt/minimal/src/runtime/core.rs

```diff
@@ -152,8 +154,10 @@ impl MinimalRuntime {
         }
     }
 
-    pub fn get_random(&self, buf: &mut [u8]) -> usize {
-        // TODO: Once the Randomness PR is in, fix this.
-        buf.len()
+    pub fn get_random(&self, buf: &mut [MaybeUninit<u8>], flags: GetRandomFlags) -> usize {
+        // rarely if ever would fail
+        // FIXME: Calling sys_get_random creates a deadlock
```

Comment:
How so?

Reply:
I fixed that sorry! I'll remove that comment. The rarely if ever would fail part is true. Basically would only fail on startup if:
a) there was no on-demand entropy sources (like jitterrand)
b) nonblocking was specified
c) there was no entropy contributed from sporadic randomness sources before trying to get randomness

Reply:
Great, yeah, I was mostly curious about the deadlock comment. The other stuff makes sense!

Reply:
fixed!